scripts/xe-syslog-reconfigure

```diff
@@ -14,32 +14,11 @@ do
   shift
 done
 
-if [ -r /etc/syslog.conf ]; then
-  conf_file=/etc/syslog.conf
-  service=syslog
-fi
-
-if [ -r /etc/rsyslog.conf ]; then
-  conf_file=/etc/rsyslog.conf
-  service=rsyslog
-fi
-
-if [ -r /etc/rsyslog.d/xenserver.conf ]; then
-  conf_file=/etc/rsyslog.d/xenserver.conf
-  service=rsyslog
-fi
-
-if [ -z "$service" ]; then
-  echo "Error: unable to determine syslog service" >&2
-  exit 1
-fi
 
-sed -e '/^\*\.\*.*[@~]/ d' $conf_file >/etc/syslog.$$
 if [ $remote -eq 1 ]; then
-  echo -e "*.* @$host\n*.* ~" >>/etc/syslog.$$
+  echo "*.* @$host" >/etc/rsyslog.d/remote.conf
```

Comment:
Can a comment be added to `/etc/rsyslog.d/remote.conf` explaining that it's an autogenerated file created by `xe-syslog-reconfigure`?